sequel/Gemfile

```diff
@@ -0,0 +1,29 @@
+source 'https://rubygems.org'
+
+module ::Kernel
+  def sequel_master?
+    ENV["SEQUEL_MASTER"] == '1'
+  end
+
+  def mysql2_prepared_statements?
+    ENV["MYSQL2_PREPARED_STATEMENTS"] == '1'
+  end
+end
+
+if sequel_master?
+  gem 'sequel', github: 'jeremyevans/sequel', branch: 'master'
+else
+  gem 'sequel', path: '/sequel'
+end
+
+gem 'rails', '~> 5.0.0.rc1'
+gem 'sqlite3', '1.3.10'
```

Comment:
If we're running it locally, we should expect the user to have MySQL and PG set up.